packages/inter-protocol/test/smartWallet/contexts.js

```diff
@@ -125,3 +127,30 @@ export const makeDefaultTestContext = async (t, makeSpace) => {
     simpleCreatePriceFeed,
   };
 };
+
+/**
+ * @param {Awaited<ReturnType<typeof coalesceUpdates>>} state
+ * @param {Brand<'nat'>} brand
+ */
+export const purseBalance = (state, brand) => {
+  const balances = Array.from(state.balances.values());
+  const match = balances.find(b => b.brand === brand);
+  if (!match) {
+    console.debug('balances', ...balances);
+    assert.fail(`${brand} not found in record`);
+  }
+  return match.value;
+};
+/**
+ * @param {import('@agoric/smart-wallet/src/smartWallet.js').CurrentWalletRecord} record
+ * @param {Brand<'nat'>} brand
+ */
+export const currentPurseBalance = (record, brand) => {
```

Comment:
likewise `currentInvitationPurseBalance`?

Reply:
If you really do need the balance of some other brand, use the relevant bank directly, as in:

https://github.com/Agoric/agoric-sdk/blob/adc0836e01eab0fd6084ecf3f59687668a98ed54/packages/inter-protocol/test/smartWallet/contexts.js#L69-L70